changelog/6447.bugfix.1.rst

```diff
@@ -0,0 +1 @@
+Fixed a bug where the private property ``_default_observer_coordinate`` for `~sunpy.map.GenericMap` that is used to specify a custom default observer in the absence of sufficient observer metadata was being used even when there was sufficient observer metadata.
```

Comment:
```suggestion
Fixed a bug where the private property ``_default_observer_coordinate`` for `~sunpy.map.GenericMap` was being used even when there was sufficient observer metadata in the header.
```
Otherwise that was a very long sentence

Reply:
Sure